src/coreclr/jit/importercalls.cpp

```diff
@@ -9203,7 +9210,7 @@ void Compiler::impCheckCanInline(GenTreeCall*           call,
 
         // Return type
         //
-        var_types const fncRetType = pParam->call->TypeGet();
+        var_types const fncRetType = genActualType(pParam->call->gtReturnType);
```

Comment:
Now that we're cleaning up I'm not sure that it makes much sense for us to store this... It's redundant data. Can we just delete `InlineCandidateInfo::fncRetType` in favor of `inlineInfo->methInfo.args.retType`? I see one place that can switch to  a check against `CORINFO_TYPE_VOID` and another place that can use `genActualType(JITtype2varType(inlineInfo->methInfo.args.retType))` directly.